README.md

```diff
@@ -38,23 +38,21 @@ The great thing about Machine ID is that it can helpe with use cases outside of
 * A Github Personal Access Token (If you plan to run this action from a private github repo).
 * AWS Credentials for Terraform provisioning (If you plan to run the Terraform step in this workflow)
 
-
-
 ### Teleport Configuration
-1. Create a Github token in Teleport, explicitly naming the repository where you will run this action. This will enable trust between the repository and your Teleport cluster. An example token definition is in `teleport/gha_token.yaml`. Apply this file to your cluster using `tctl create -f gha_token.yaml`
+1. Create a Github token in Teleport, explicitly naming the repository where you will run this action. This will enable trust between the repository and your Teleport cluster. An example token definition is in `teleport/gha_token.yaml`. Apply this file to your cluster using `tctl create -f gha_token.yaml`. 
 2. Create a role for your Machine ID bot. This bot should have permissions to only manage tokens and nothing else. An example of a role with these privileges is at `teleport/gha_bot_role.yaml`. Apply the role to your cluster using `tctl create -f gha_bot_role.yaml`
 3. Create the Machine ID bot. This will link a bot user to the role and token we just created. `tctl bots add gha-token-bot --roles=<match your role name> --token=<match your token name>`
 
 ### Github Actions Configuration
 1. The example workflow at `.github/workflows/main.yaml` requires four secrets to run. 
-   * PAT:  A github Personal Access Token which will give the workflow permissions to check out your repository if it is private. 
+   * PAT:  A github Personal Access Token which will give the workflow permissions to check out your repository if it is private. Even if your repo is public you will still need this token for the workflow to run properly.
```

Comment:
Lets just make this:
A github Personal Access Token which will give the workflow permissions to check out your repository.